sklearn/preprocessing/imputation.py

```diff
@@ -112,9 +112,14 @@ class Imputer(BaseEstimator, TransformerMixin):
     ----------
     statistics_ : array of shape (n_features,)
         The imputation fill value for each feature if axis == 0.
+        The values corresponding to features ``all_missing_` is ``nan``.
```

Comment:
Sorry for the delay. Thanks will reword it. You are right about the statistic being ``nan`` when all are missing. Even if there is one value for the feature, the corresponding statistic becomes the value itself. I am using ``all_missing_`` to keep a track of features with all missing values during fit, so that it is possible to compare with the input to transform and raise a RunTime Warning accordingly. Please do let me know if there is a better way to achieve that.